.circleci/continue_config.yml

```diff
@@ -352,6 +352,14 @@ jobs:
             cd presto-native-execution
             ./scripts/setup-conda.sh
       - run:
+          # Doesn't build with -DPRESTO_ENABLE_PARQUET=ON because this requires
+          # arrow_ep, which raises deprecation warnings when compiled with
+          # clang. These warnings cannot be disabled from presto because
+          # arrow_ep is defined via ExternalProject_Add. To fix this, add
+          # -Wno-deprecated-declarations to DISABLED_WARNINGS or CMAKE_CXX_FLAGS
+          # in presto's CMakeLists.txt, then patch ExternalProject_Add where
+          # arrow_ep is defined, by adding -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
+          # to ARROW_CMAKE_ARGS.
```

Comment:
This can be done with a patch, where submodule is updated in the Makefile (only on macOS)